external_push/models.py

```diff
@@ -382,13 +383,26 @@ class Meta:
         (60 * 60 + 1, '1 hour'),
     )
 
+    DEVICE_GRAVITY = 'gravity'
+    DEVICE_BREWPI = 'brewpi'
+
+    DEVICE_CHOICES = (
+        (DEVICE_GRAVITY, "Gravity sensors"),
+        (DEVICE_BREWPI, "Brewpi sensors"),
+    )
+
     status = models.CharField(max_length=24, help_text="Status of this push target", choices=STATUS_CHOICES,
                               default=STATUS_ACTIVE)
     push_frequency = models.IntegerField(choices=PUSH_FREQUENCY_CHOICES, default=60 * 15,
                                          help_text="How often to push data to the target")
     logging_url = models.CharField(max_length=256, help_text="Brewfather Logging URL", default="")
 
-    gravity_sensor_to_push = models.ForeignKey(to=GravitySensor, related_name="brewfather_push_target", on_delete=models.CASCADE,
+    device_type = models.CharField(max_length=24, help_text="What type of device to send", choices=DEVICE_CHOICES,
+                              default=DEVICE_GRAVITY)
+    brewpi_to_push = models.ForeignKey(to=BrewPiDevice, related_name="brewfather_push_target2", null=True, blank=True, on_delete=models.CASCADE,
```

Comment:
Something feels off about using models.CASCADE here, but I  need to poke at this branch a bit to figure out what the alternative would be...

Reply:
I just copied that from the previous code and I'm not really that experienced in Django so feel free to make it better.